VMEncryption/main/Common.py

```diff
@@ -60,9 +60,17 @@ class CommonVariables:
     """
     CVM LUKS2 header token
     """
+    #this token id is used to store Primary ADE (encryption setting + wrapped passphrase) token.
     cvm_ade_vm_encryption_token_id = 5
+    #this token id is used to store backup of token id 5, during KEK rotation.
+    cvm_ade_vm_encryption_backup_token_id = 6
     ADEEncryptionVersionInLuksToken_1_0='1.0'
-    PassphraseNameValue = 'LUKSPasswordProtector'
+    PassphraseNameValueProtected = 'LUKSPasswordProtector'
```

Comment:
As we discussed in sync meeting, let's consider rotating the pass phrase as well.